scripts/utils/verify_scripts.js

```diff
@@ -11,20 +11,40 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
   )
   const { getMasterCopy, getFallbackHandler } = require("./internals")(web3, artifacts)
   const { getDexagPrice, checkNoProfitableOffer } = require("./price_utils")(web3, artifacts)
+  const { equalUpToOrder } = require("./js_helpers")
 
   const GnosisSafe = artifacts.require("GnosisSafe.sol")
   const GnosisSafeProxy = artifacts.require("GnosisSafeProxy.sol")
   const gnosisSafeMasterCopy = GnosisSafe.deployed()
 
   const pageSize = 50
 
-  const verifyBracketsWellFormed = async function(masterAddress, bracketAddresses, logActivated = false) {
+  const verifyBracketsWellFormed = async function(
+    masterAddress,
+    bracketAddresses,
+    masterThreshold,
+    masterOwners,
+    logActivated = false
+  ) {
     const log = logActivated ? (...a) => console.log(...a) : () => {}
 
     const gnosisSafe = await gnosisSafeMasterCopy
     const master = await getSafe(masterAddress)
     const brackets = await Promise.all(bracketAddresses.map(bracketAddress => getSafe(bracketAddress)))
 
+    if (!masterOwners || !masterThreshold) log("- Master safe owner verification skipped")
+    else {
+      log("- Verify owners of masterSafe")
+      const threshold = master.getThreshold()
+      const owners = master.getOwners()
+      assert.equal(
+        await threshold,
+        masterThreshold,
+        "Master threshold is " + (await threshold) + " while it is supposed to be " + masterThreshold
+      )
+      assert(equalUpToOrder(await owners, masterOwners), "Master owners are different than expected")
```

Comment:
In order to save some code, would here also work something like this:
```
assert.isDeepEqual((await owners).sorted, masterOwners.sorted, "")
```
?

Reply:
Very nice, this is much cleaner than creating my own comparison function.